trinity/utils/chains.py

```diff
@@ -137,6 +137,9 @@ def construct_chain_config_params(args):
     if args.sync_mode is not None:
         yield 'sync_mode', args.sync_mode
 
+    if args.max_peers is not None:
+        yield 'max_peers', int(args.max_peers)
```

Comment:
This `int` cast shouldn't be necessary as it should be done as part of the argument parsing.

Reply:
If I don't cast this, then I will be getting the following:

```
{'max_peers': '10',
 'network_id': 1,
 'port': 30303,
 'preferred_nodes': None,
 'sync_mode': 'full'}

  File "/Users/glaksmono/Documents/py-evm/p2p/server.py", line 101, in __init__
    if self.max_peers < self.min_peers:
TypeError: '<' not supported between instances of 'str' and 'int'
```